src/components/FeaturedContent/FeaturedContent.test.tsx

```diff
@@ -0,0 +1,221 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import renderer from "react-test-renderer";
+
+import FeaturedContent from "./FeaturedContent";
+import Button from "../Button/Button";
+import Heading from "../Heading/Heading";
+import Text from "../Text/Text";
+export const textContent = (
+  <>
```

Comment:
Nitpick: you don't technically need the react fragment in this case because this is all wrapped in a div.